packages/integration-sdk-cli/src/utils/diff.ts

```diff
@@ -0,0 +1,84 @@
+import fs from "fs";
+import { diff as diffJson } from 'json-diff'
+
+export function findDifferences(oldJsonDataPath, newJsonDataPath, logOnlyKeyChanges) {
+
+  const neo = JSON.parse(fs.readFileSync(newJsonDataPath, 'utf8'))
```

Comment:
Just curious - why use `neo` and not `new`?

Reply:
New is a keyword. Couldn't think of a better word at the time.

Reply:
👍